README_JP.md

```diff
@@ -0,0 +1,317 @@
+English(README.md) | [简体中文](README_CN.md) | [Deutsch](README_GE.md) | 日本語
```

Comment:
Please also make English README.md as a link in this line by -- 
```
[English](README.md) | [简体中文](README_CN.md) | [Deutsch](README_GE.md) | 日本語
```

Reply:
Ok. I will add your requirements and pull request again.

Reply:
I added the link to Japanese README file in other README_XX.md files.

Reply:
Please also add the link to English README file in your `README_JP.md` file.

Reply:
Added :)